packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -77,16 +80,24 @@ export const MafsGraph = (props: MafsGraphProps) => {
                 <View
                     className="mafs-graph"
                     style={{
-                        height,
-                        width: "intrinsic",
                         position: "relative",
                         padding: "25px 25px 0 0",
                         boxSizing: "content-box",
                         marginLeft: "20px",
                         marginBottom: "20px",
                         pointerEvents: props.static ? "none" : "auto",
                         userSelect: "none",
+                        width,
+                        height,
+                    }}
+                    onKeyUp={(event) => {
+                        if (event.key === "Backspace") {
+                            dispatch(actions.global.deleteIntent());
+                            graphRef.current?.focus();
+                        }
```

Comment:
I'm still a little confused about why we have to handle this event on the graph component rather than on the point itself. I tried adding an `onKeyDown` handler to the `g[data-testid='movable-point']` element and it works in both Edge and Safari. The focused point's event handler is fired when you press a key.

The advantage of putting the event handler on the point would be that we wouldn't have to keep track of which point is focused in the useReducer state.

Am I missing something?

Reply:
I'm definitely open to changing it but there are a couple of things I like:

1. This lets us add a delete intent to other kinds of graphs with no fuss. I think we'll use that for unlimited polygon (but that's a gut feeling).
2. This makes it natural to add other keyboard shortcuts and gives a natural place for them to live
3. I think I need to know the selected point regardless of this change. After a point is added we want to focus on the newly added point. So as a learner I tab to the "Add point" button, hit space, the point is selected and I can manipulate it with my keyboard. I think that means we'll need to keep track of the selected point so that after the add button is activated we select the right thing.